templates/commands/goldentest/new_test_flags.go

```diff
@@ -112,4 +100,16 @@ func (r *NewTestFlags) Register(set *cli.FlagSet) {
 		}
 		return nil
 	})
+
+	// Default Location to the second CLI argument, if given
+	// If not given, default to current directory.
+	set.AfterParse(func(existingErr error) error {
+		r.Location = strings.TrimSpace(set.Arg(1))
+
+		if r.Location == "" {
```

Comment:
Is there a way to test this? Some examples of flag testing are in abc_test.go and commands/render/render_test.go.

Reply:
done